scripts/reset-dockstoretestuser4-resources.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+STACK=$1
+
+
+if [ "$STACK" == "qa" ]
+then
+  URL="https://qa.dockstore.org"
+  TOKEN=${CYPRESS_QA_TOKEN}
+elif [ "$STACK" == "staging" ]
+then
+  URL="https://staging.dockstore.org"
+  TOKEN=${CYPRESS_STAGING_TOKEN}
+else
+  URL="https://dockstore.org"
+  TOKEN=${CYPRESS_PROD_TOKEN}
+fi
+
+
+USER=$(curl -X 'GET' \
+  "${URL}/api/users/username/DockstoreTestUser4" \
+  -H 'accept: application/json')
+
+USER_ID=$(echo "$USER" | jq -r ".id")
+
+while IFS=$'\n' read -r line
```

Comment:
Not a BASH expert nor do I know its best practices that well, but I would still suggest adding comments and/or creating functions, e.g., something like `getUserPublishedWorkflows`.